files/build_templates/sonic_debian_extension.j2

```diff
@@ -770,6 +770,15 @@ if [ "${PIPESTATUS[0]}" != "0" ]; then
     exit 1
 fi
 
+sudo mkdir -p $FILESYSTEM_ROOT/$SONIC_PACKAGE_MANAGER_FOLDER/manifests
+target_machine="$TARGET_MACHINE" j2 $BUILD_TEMPLATES/default_manifest.j2 | sudo tee $FILESYSTEM_ROOT/$SONIC_PACKAGE_MANAGER_FOLDER/manifests/default_manifest
+if [ "${PIPESTATUS[0]}" != "0" ]; then
+    echo "Failed to generate default_manifest" >&2
+    exit 1
+fi
+
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install paramiko
```

Comment:
Why install new package?

Reply:
If required for sonic-utilities then specify in setup.py - https://github.com/sonic-net/sonic-utilities/blob/master/setup.py#L230

Reply:
I see paramiko is already present in setup.py.
Hence removed it from here now.